### PR TITLE
Support for Sindre Sorhus

### PR DESCRIPTION
**Pull request by znarf**
First support for Sindre Sorhus collective.

- http://github.com/sindresorhus
- https://opencollective.com/sindresorhus

Screen Shot 2019-03-12 at 12 02 26

cc @sindresorhus!

### CODE REVIEW

File: src/scripts/update-collectives.js [hunk 1]
```diff
@@ -72,6 +78,39 @@ async function getCollectiveRepos(github) {
...
+              }
+            }
+          }
+          if (reposForUser.length < reposListForUserParameters.per_page) {
+            break;
+          }
+          reposListForUserParameters.page++;
+        }
+      } catch (e) {
+        logger.warn(`Could not fetch repos for user ${githubUser}`);
```
Comment (sindresorhus) [inline src/scripts/update-collectives.js]: You should probably log the error too.

**Comment by sindresorhus** [comment]
@znarf Thanks for doing this 🙌

**Comment by sindresorhus** [comment]
`Used In:` needs a different wording here. Maybe `Author's Packages Used In:`?

**Comment by znarf** [comment]
@sindresorhus Yes, the copy is not he best, but you're currently an edge case ;-) I was thinking maybe first listing all the packages ```Packages: awesome, p-queue, ...`. There is not always the space to display that.

Also, the following copy "Projects Requiring Funding" is not adapted.

**Pull request merged**